pages/common/bash.md

```diff
@@ -28,6 +28,10 @@
 
 `bash -s`
 
+- Stop argument processing bash and pass all the following arguments directly to the command:
+
+`bash -c {{ls}} -- -lA`
```

Comment:
We've already got a `bash -c` example above. Perhaps this could be just a regular `"{{path/to/script.sh}}"` instead?

Reply:
I think you might have missed this comment @KsRyY?

Reply:
yes. making the changes.